SCons/Tool/applelink.py

```diff
@@ -212,10 +165,21 @@ def generate(env):
     # pre/suffixes:
     env['LDMODULEPREFIX'] = ''
     env['LDMODULESUFFIX'] = ''
-    env['LDMODULEFLAGS'] = SCons.Util.CLVar('$LINKFLAGS -bundle')
-    env['LDMODULECOM'] = '$LDMODULE -o ${TARGET} $LDMODULEFLAGS $SOURCES $_LIBDIRFLAGS $_LIBFLAGS $_FRAMEWORKPATH $_FRAMEWORKS $FRAMEWORKSFLAGS'
-
-    env['__SHLIBVERSIONFLAGS'] = '${__libversionflags(__env__,"SHLIBVERSION","_SHLIBVERSIONFLAGS")}'
+    env['LDMODULEFLAGS'] = CLVar('$LINKFLAGS -bundle')
+    env[
```

Comment:
that's an awkward looking break after the opening bracket. Plus, string pasting works without needing the backslash

Reply:
Autoformatter did that..

Reply:
fixed.